### DISCUSSION TITLE
Not Supported in AWS Amplify

### DISCUSSION THREAD

**Discussion by shumakmanohar** [amannn/next-intl#942] [Q&A]
### Description

Locale Not working in AWS Amplify ? 
Any workarounds

### Mandatory reproduction URL

Used App Router

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: … 

### Expected behaviour

Locale is not detected , but locale  works in vercel

**Comment by amannn** [comment]
I'm moving this to a discussion since it's a usage question. I unfortunately don't have experience with AWS Amplify.

**Reply by shumakmanohar** [reply]
Sorry Amann , for raising this , It was a mistake on my side , It did work for after changing some middleware , no issue with  the next-intl library.

**Reply by amannn** [reply]
Great to hear you were able to figure it out! 🙌
